### PR TITLE
LOG-2819: Extract log level from structured if exist

### PR DESCRIPTION
**Pull request by vparfonov**
Signed-off-by: Vitalii Parfonov <vparfono@redhat.com>### Description

/cc 
/assign 

/cherry-pick 

### Links

- Depending on PR(s):
- Bugzilla:
- Github issue:
- JIRA:
- Enhancement proposal:

### CODE REVIEW

File: internal/generator/fluentd/pipeline_to_output.go [hunk 1]
```diff
@@ -39,6 +39,17 @@ const (
   </parse>
 </filter>
 {{end}}`
+
+	LogLevelSetterTemplate = `
+# {{.Desc}}
+<filter **>
+    @type record_modifier
+    <record>
+       _dummy_ ${struct_level = record.dig('structured','level'); level = record['level']; if ![nil, level].include?(struct_level) && [nil, 'unknown'].include?(level); record['level']=struct_level; end; nil}
```
Comment (vimalk78) [inline internal/generator/fluentd/pipeline_to_output.go:48]: the default level we decided to use since GCP is `default`

Comment (vparfonov) [reply internal/generator/fluentd/pipeline_to_output.go:48]: `unknown` -> `default` ?

Comment (vimalk78) [reply internal/generator/fluentd/pipeline_to_output.go:48]: yes

**Pull request closed without merging**